src/Framework/TestResult.php

```diff
@@ -1011,6 +1016,14 @@ public function stopOnSkipped(bool $flag): void
         $this->stopOnSkipped = $flag;
     }
 
+    /**
+     * Enables or disables the stopping for any defect: skipped, error, failure, warning, incomplete or risky.
```

Comment:
I believe this description is not up to date now

Reply:
Fixed